src/main/java/com/aetherteam/aether_genesis/data/generators/loot/GenesisBlockLoot.java

```diff
@@ -26,6 +26,9 @@ public void generate() {
         this.dropSelfDouble(GenesisBlocks.GREEN_AERCLOUD.get());
         this.dropSelfDouble(GenesisBlocks.PURPLE_AERCLOUD.get());
         this.dropSelfDouble(GenesisBlocks.STORM_AERCLOUD.get());
+        this.dropSelfDouble(GenesisBlocks.HOLYSTONE_HEADSTONE.get());
```

Comment:
The holystone variants should just use this.dropSelf, they don't have double drops.

Reply:
got it